lib/procfile.js

```diff
@@ -33,27 +32,28 @@ function readProcFile (pid, options, done) {
       stime: parseFloat(infos[12]),
       cutime: parseFloat(infos[13]),
       cstime: parseFloat(infos[14]),
-      start: parseFloat(infos[19]) / cpu.clockTick,
+      start: parseFloat(infos[19]) / cpuInfo.clockTick,
       rss: parseFloat(infos[21]),
-      uptime: cpu.uptime
+      uptime: cpuInfo.uptime
     }
 
-    // http://stackoverflow.com/questions/16726779/total-cpu-usage-of-an-application-from-proc-pid-stat/16736599#16736599
+    var memory = stat.rss * cpuInfo.pageSize
+
+    // https://stackoverflow.com/a/16736599/3921589
     var childrens = options.childrens ? stat.cutime + stat.cstime : 0
-    var total = (stat.stime - (hst.stime || 0) + stat.utime - (hst.utime || 0) + childrens) / cpu.clockTick
+    // process usage since last call in seconds
+    var total = (stat.stime - (hst.stime || 0) + stat.utime - (hst.utime || 0) + childrens) / cpuInfo.clockTick
     // time elapsed between calls in seconds
     var seconds = Math.abs(hst.uptime !== undefined ? stat.uptime - hst.uptime : stat.start - stat.uptime)
-    if (seconds === 0) seconds = 1 // we sure can't divide through 0
+    var cpu = seconds > 0 ? (total / seconds) * 100 : 0
 
     history.set(stat, options.maxage)
 
-    var cpuPercent = Math.min(Math.round((total / seconds) * 100000) / 1000, 100.0)
-    var memory = stat.rss * cpu.pageSize
 
```

Comment:
There's a linting issue actually :)
```
  C:\projects\pidusage\lib\procfile.js:51:1: More than 1 blank line not allowed.
```

I think that appveyor will be happy after!

Reply:
Yes I saw it, actually there was a bug in the test command.